csrc/core/device.h

```diff
@@ -67,7 +68,7 @@ class Device {
   constexpr explicit Device(int platform_id, int device_id = 0)
       : platform_id_(platform_id), device_id_(device_id) {}
 
-  explicit Device(const char *platform_name, int device_id = 0);
+  MMDEPLOY_API explicit Device(const char *platform_name, int device_id = 0);
```

Comment:
Why not mark `class Device` as `class MMDEPLOY_API Device`?

Reply:
it's the only function with its implmentation non-inlined